snc-library.sh

```diff
@@ -253,3 +253,52 @@ function delete_operator() {
         # Wait until the operator pod is deleted before trying to delete the resources it manages
         ${OC} wait --for=delete pod/${pod} --timeout=120s -n ${namespace} || ${OC} delete pod/${pod} --grace-period=0 --force -n ${namespace} || true
 }
+
+function operators_available() {
+        # Check the cluster operator output, status for available should be true for all operators
+        ${OC} get co -ojsonpath='{.items[*].status.conditions[?(@.type=="Available")].status}' | grep -v False
+}
+
+function operators_progressing() {
+        # Check the cluster operator output, status for progressing should be false for all operators
+        ${OC} get co -ojsonpath='{.items[*].status.conditions[?(@.type=="Progressing")].status}' | grep -v True
+}
+
+function operators_degraded() {
+        # Check the cluster operator output, status for available should be false for all operators
+        ${OC} get co -ojsonpath='{.items[*].status.conditions[?(@.type=="Degraded")].status}' | grep -v True
+}
+
+function wait_till_cluster_stable() {
+	sleep 1m
+
+        local retryCount=30
+        local numConsecutive=3
```

Comment:
indentation is not the same for `sleep` and `local`? Why start with a sleep when we have more sophisticated logic for waiting?